fixtures/fovitec/600xb.json

```
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "600XB",
+  "categories": ["Other"],
+  "meta": {
+    "authors": ["Aninymous"],
+    "createDate": "2021-08-03",
+    "lastModifyDate": "2021-08-03"
+  },
+  "links": {
+    "productPage": [
+      "https://www.fovitec.com/products/600xb-bi-color-led-panel-with-dmx"
+    ]
+  },
+  "availableChannels": {
+    "Intensity": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Color Temperature": {
+      "capability": {
+        "type": "ColorTemperature",
+        "colorTemperatureStart": "cold",
+        "colorTemperatureEnd": "warm"
+      }
+    }
+  },
+  "modes": [
+    {
+      "name": "600XB",
```

```suggestion
      "name": "Default",
```
I thought the standard if there is only one mode is `Default` ? Could be wrong though

---

If there is no explicit name in the manual, I prefer this:

```suggestion
      "name": "2-channel",
      "shortName": "2ch",
```